Dockerfiles/cluster-agent/README.md

```diff
@@ -8,12 +8,13 @@ The following environment variables are supported:
 
 - `DD_API_KEY` - **required** - your [Datadog API key][https://app.datadoghq.com/account/settings#api].
 - `DD_HOSTNAME`: hostname to use for the DCA.
-- `DD_CLUSTER_AGENT_CMD_PORT`: Port for the DCA to serve, default is `5005`.
-- `DD_USE_METADATA_MAPPER`: Enables the cluster level metadata mapping, default is `true`.
+- `DD_CLUSTER_AGENT_CMD_PORT`: port for the DCA to serve, default is `5005`.
+- `DD_USE_METADATA_MAPPER`: enables the cluster level metadata mapping, default is `true`.
```

Comment:
I'm confused, if this `README` is specific to the DCA, how come we have agent config options in here like `DD_USE_METADATA_MAPPER`?